### ISSUE TITLE
When locking Ubuntu ( Win + L ), wait till screen is suspended, the Dock will occur twice on the screen

### ISSUE THREAD

**Issue by guenter-r** [micheleg/dash-to-dock#1282]
After I locked ubuntu and waited till the screen is dark, I woke the laptop again to enter my login credentials. 
At this point already, there is a dock on the lockscreen. 
When logged back in there are pretty much two docks on the screen:

**Comment by monocongo** [comment]
This is also happening for me on Ubuntu 20.04. I am using gdm3 with Wayland enabled. If I can provide further details that may be useful then please request these details in a follow up comment on this issue.

Finally, a big THANKS to the authors/contributors of this extension, it really makes a difference in my desktop experience, I really appreciate your efforts!

**Comment by monocongo** [comment]
I seem to have partially resolved this issue by installing the Extensions app and then disabling Ubuntu Dock. Apparently you get bad juju if you run Ubuntu Dock and Dash to Dock simultaneously. After disabling Ubuntu Dock now when I lock the screen I still see the icons along the bottom but when I unlock the screen I no longer see the duplicated icons in the dock. So the main issue has been resolved, or it appears to have been. If things improve even further (or devolve) after a reboot I will update this issue accordingly.

This [helpful page](https://itsfoss.com/disable-ubuntu-dock/) walked me through the process.

TL;DR:
```
$ sudo apt install gnome-shell-extensions
$ sudo gnome-extensions disable ubuntu-dock@ubuntu.com
```

**Comment by guenter-r** [comment]
Thanks @monocongo, this works for me. When locking the first time its all fine, when locking a second time or more often, the dock remains on the lockscreen - the duplicate dock is gone though!

**Comment by haogroot** [comment]
> I seem to have partially resolved this issue by installing the Extensions app and then disabling Ubuntu Dock. Apparently you get bad juju if you run Ubuntu Dock and Dash to Dock simultaneously. After disabling Ubuntu Dock now when I lock the screen I still see the icons along the bottom but when I unlock the screen I no longer see the duplicated icons in the dock. So the main issue has been resolved, or it appears to have been. If things improve even further (or devolve) after a reboot I will update this issue accordingly.
> 
> This [helpful page](https://itsfoss.com/disable-ubuntu-dock/) walked me through the process.
> 
> TL;DR:
> 
> ```
> $ sudo apt install gnome-shell-extensions
> $ sudo gnome-extensions disable ubuntu-dock@ubuntu.com
> ```

Thanks @monocongo , this also works for me on Ubuntu 20.04.1 !!!

**Comment by anonymouse64** [comment]
For me, when this bug happens, the icon that causes the duplication is always the "Trash" icon, it is normally present in the normal dock, but the duplicated dock that appears never has the "Trash" on it and so it shows up as the images here show. 

I'm also on Ubuntu 20.04 with stock gnome, etc.

**Comment by diningPhilosopher64** [comment]
Facing the same issue with a fresh Install of Ubuntu 20.04. Followed @monocongo 's suggestion and worked like charm.

Cheers!
